lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +240,41 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        result_list = []
+        sentence_list = split_by_sentence(text)
+
+        for position, element in enumerate(sentence_list):
+            conllu_tokens = []
+            words_in_sentence = element.split()
```

Comment:
where is mystem initialization and call?

Reply:
done